src/workers/newNotificationV2Mail.ts

```diff
@@ -880,6 +882,27 @@ const notificationToTemplateData: Record<NotificationType, TemplateDataFunc> = {
       gifter_image: gifter.image,
     };
   },
+  user_received_award: async (con, user, notification) => {
+    const transaction = await con.getRepository(UserTransaction).findOneOrFail({
+      where: {
+        id: notification.referenceId,
+        receiverId: user.id,
+      },
+    });
+
+    const sender = await transaction.sender;
+    const product = await transaction.product;
+
+    const coreAmount = Intl.NumberFormat('en-US').format(transaction.value);
```

Comment:
Let's copy `formatCoresCurrency` from frontend, so it does K formatting over X amount, just to bulletproof for the future.

Reply:
Done